etc/p3-config/p3-config.yml

```diff
@@ -603,8 +603,7 @@ p3_image_mlnx_ofed:
     - "vm"
     - "systemd-modules-load"
   env:
-    DIB_MLNX_OFED_VERSION: "4.1-1"
```

Comment:
export MLNX_OFED_VERSION=${DIB_MLNX_OFED_VERSION:?"A revision of OFED is required"}

Reply:
Now, if MLNX_OFED_REPO_FILE is specified, this is overridden.